components/component-stylesheets/modal.css

```diff
@@ -0,0 +1,34 @@
+.modal-container {
+    width: 50rem;
+    height: 30rem;
+    border-radius: 1rem;
+    background-color: var(--mr-warning-alert-bg-color);
+    display: flex;
```

Comment:
suggestion:- flex, align-item,justify-contentyou can also put this property in the utils file , since it is used  many times